public/Aframe/samples/loading-scene-1/scene-1.html

```diff
@@ -0,0 +1,117 @@
+        <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.0/dist/aframe-extras.min.js"></script>
+        <script src="//cdn.rawgit.com/donmccurdy/aframe-physics-system/v4.0.1/dist/aframe-physics-system.min.js"></script>
+        <script src="https://unpkg.com/aframe-click-drag-component"></script>
+        <script src="components.js"></script>
+    </head>
+    <body>
+        <a-scene
+            id="scene"
+            background="color: #ECECEC"
+            physics="debug: false; gravity: 0"
+            gravity-switcher
+        >
+            <a-assets>
+                <img id="space" src="assets/stars.jpeg" />
+            </a-assets>
+
+            <a-entity id="container"></a-entity>
+
+            <a-box
+                src="#space"
+                static-body
+                position="0 0 -20"
+                width="40"
+                height="30"
+                depth="5"
+                color="#a4f542"
+            ></a-box>
+            <a-box
+                src="#space"
+                static-body
+                position="0 0 20"
+                width="40"
+                height="30"
+                depth="5"
+                color="#42a4f5"
+            ></a-box>
+            <a-box
+                src="#space"
+                static-body
+                position="20 0 0"
+                width="5"
+                height="30"
+                depth="40"
+                color="#f56042"
+            ></a-box>
+            <a-box
+                src="#space"
+                static-body
+                position="-20 0 0"
+                width="5"
+                height="30"
+                depth="40"
+                color="#6642f5"
+            ></a-box>
+            <a-box
+                src="#space"
+                static-body
+                position="0 17.5 0"
+                width="40"
+                height="5"
+                depth="40"
+                color="#fcba03"
+            ></a-box>
+            <a-box
+                src="#space"
+                static-body
+                position="0 -17.5 0"
+                width="40"
+                height="5"
+                depth="40"
+                color="#fcba03"
+            ></a-box>
+
+            <a-box
+                static-body
+                position="0 -10 -8"
+                width="3"
+                height="10"
+                depth="3"
+                color="#ffffff"
+            ></a-box>
+            <a-cylinder
+                static-body
+                position="0 -5 -8"
+                radius="1.25"
+                height="1"
+                color="#ff1500"
+                detect-button
+                add-ball
+            ></a-cylinder>
+
+            <a-entity id="rig" position="0 0 0">
+                <a-entity
+                    id="camera"
+                    camera
+                    camera-floor
```

Comment:
`camera-floor` is a leftover from another example I believe, so we can remove this